jpsreport/methods/PedData.cpp

```diff
@@ -67,17 +66,7 @@ bool PedData::ReadData(
         "INFO:\tfull name of the trajectory is: <%s>", fullTrajectoriesPathName.string().c_str());
     bool result = true;
     if(trajformat == FORMAT_XML_PLAIN) {
-        TiXmlDocument docGeo(fullTrajectoriesPathName.string());
-        if(!docGeo.LoadFile()) {
-            Log->Write("ERROR: \t%s", docGeo.ErrorDesc());
-            Log->Write(
-                "ERROR: \tcould not parse the trajectories file <%s>",
-                fullTrajectoriesPathName.string().c_str());
-            return false;
-        }
-        TiXmlElement * xRootNode = docGeo.RootElement();
-        result                   = InitializeVariables(xRootNode); //initialize some global
-                                                                   //variables using xml format
+        Log->Write("WARNING: Input trajectory file for jpsreport should be in txt format");
```

Comment:
Why do you remove the xml format?

Reply:
We write in the [documentation](https://www.jupedsim.org/jpsreport_inifile.html#trajectories):

> The format of trajectory files should be `.txt` or `.xml`. 

We'll have to change that when we remove that part from the code.

Reply:
@chraibi Thanks for commit 41aac25.